.travis.yml

```diff
@@ -5,7 +5,7 @@ cache:
   directories:
     - ".meteor"
     - ".babel-cache"
-script: TEST_PACKAGES_EXCLUDE="less" ./packages/test-in-console/run.sh
+script: TEST_PACKAGES_EXCLUDE="less,blaze" ./packages/test-in-console/run.sh
```

Comment:
Not sure if this is needed, now that git submodule is fixed. Blaze is still bundled with Meteor, so it should probably be tested with it as well?